frontend/App.js

```diff
@@ -1,3 +1,4 @@
+import React from "react";
```

Comment:
Out of interest, why is this needed? Generally this has not been needed in modern react unless avoiding the direct import of things like useState.

Reply:
It told me that it was needed when using JSX